src/Components/Blazor/testassets/Wasm.Authentication.Client/Shared/RedirectToLogin.razor

```diff
@@ -0,0 +1,10 @@
+@inject NavigationManager Navigation
```

Comment:
I thought we were going to have the `RedirectToLogin` component in the package. Are you thinking of that as a future follow-up?

Reply:
https://github.com/dotnet/aspnetcore/issues/18979